docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -554,10 +556,15 @@ public void pullImage(String imageName) {
 
     public String execStart(String containerId, String... commands) {
         ExecCreateCmdResponse execCreateCmdResponse = this.dockerClient.execCreateCmd(containerId)
-                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty().withCmd(commands)
-                .exec();
-        InputStream consoleOutputStream = dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty(false).withCmd(commands)
                 .exec();
+        try {
+            dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                    .exec(new ExecStartResultCallback(System.out, System.err)).awaitCompletion();
+        } catch (InterruptedException e) {
+            return "";
+        }
+        InputStream consoleOutputStream = dockerClient.copyArchiveFromContainerCmd(containerId, "/execStartTest.log").exec();
```

Comment:
I have seen in the examples of docker-java 3 that now you need to get the `execStartTest.log`. I am not sure the implications that this going to have :(. WhatI have seen in the examples is that you need to call `touch` for creating the file (https://github.com/docker-java/docker-java/blob/9ec46f44374c434a9b6132cd8611e2c8e80b8c69/src/test/java/com/github/dockerjava/netty/exec/ExecStartCmdExecTest.java#L86) but you have not called, maybe it is not mandatory at all. Have you run the ftests to see if everything works as expected?